### ISSUE TITLE
Remove tar.gz from root

### ISSUE THREAD

**Issue by varnerac** [esl/mongooseim-docker#16]
When starting a MongooseIM image using:

```
docker run -d -t -h mongooseim-1 --name mongooseim-1 -p 5222:5222 mongooseim/mongooseim:latest
```

I noticed that the release archive was still at the root of the container:

```
root@mongooseim-1:/# ls -la mongooseim.tar.gz 
-rw-rw-r-- 1 root root 41676824 Apr 11 16:02 mongooseim.tar.gz
```

It seems like you could get delete it after decompressing it, reducing the size of the container.

**Comment by michalwski** [comment]
Hi @varnerac 

Thanks for pointing this out. It's good idea!

**Comment by michalwski** [comment]
This has been solved by #28
